.travis.yml

```diff
@@ -77,7 +77,9 @@ matrix:
 
     - os: linux
       compiler: "gcc-6-release-i686"
-      env: TARGET_ARCH='i686' CCOMPILER='gcc-6' CXXCOMPILER='g++-6' BUILD_TYPE='Release'
+      env: >
```

Comment:
Before reading through the 80 pdf Yaml spec (again), I guess this just allows multi-line env vars?